aiohttp/web_ws.py

```diff
@@ -37,6 +38,8 @@ def __init__(self, *,
         self._timeout = timeout
         self._autoclose = autoclose
         self._autoping = autoping
+        self._web_socket_ready = namedtuple('web_socket_ready',
```

Comment:
Sorry, you did't get my point.
Push
`WebSocketReady = namedtuple(`WebSocketReady', ...)` at module namespace.